### DISCUSSION TITLE
Would like to implement copy to clipboard functionality for search results records with using npm react, any suggestions please

### DISCUSSION THREAD

**Discussion by GangadharMFord** [microsoft-search/pnp-modern-search#2502] [Show and tell]
Hi Team, I would like to add a copy to clipboard functionality to the PnP search results (to copy the records data like URL from search results records with button option). 
I have gone through the below URL and little confused on implementation side like where to implement on same PnP Modern search parts or at extensibility or at extensibility demo solution? Can anyone help me on how to integrate this feature to the PnP modern search results?
LINK: https://www.npmjs.com/package/react-copy-to-clipboard.
thank you.

**Comment by wobba** [comment]
You need to create a custom web component which wraps the logic. https://microsoft-search.github.io/pnp-modern-search/extensibility/custom_web_component/

**Reply by GangadharMFord** [reply]
Thank you for your replay and would like to use the below npm module.
https://www.npmjs.com/package/react-copy-to-clipboard 
Have any idea how to use the existed above solution into the Custom web component and some logic please if you free. thank you again for your time to helping me in this query.

**Reply by wobba** [reply]
SPFx development and a developer who understands the scenario is what is needed. Then you would use the component in your handlebar template.
